src/sw/saturn-sw-core.js

```diff
@@ -1,4 +1,9 @@
+import * as Sentry from '@sentry/browser'
 import { Controller } from './controller.js'
 
+Sentry.init({
+    dsn: 'https://2a6fc6930efa43dd32d3e56c92c0a7d2@o4504290295349248.ingest.sentry.io/4505902296858624',
```

Comment:
can you disable it in dev like so: https://github.com/filecoin-saturn/log-ingestor/blob/34078377e32202a72cabc84325c58f9b48ea5d4c/src/sentry.js#L37